tests/resources/Harbor-Util.robot

```diff
@@ -82,68 +83,65 @@ Install Harbor Self Signed Cert
 Log Into Harbor
     [Arguments]  ${user}=%{TEST_USERNAME}  ${pw}=%{TEST_PASSWORD}
     Maximize Browser Window
-    Input Text  username  ${user}
-    Input Text  uPassword  ${pw}
-    Click button  Sign In
-    Wait Until Page Contains  Summary
-    Wait Until Page Contains  My Projects:
-    Wait Until Keyword Succeeds  5x  1  Page Should Contain Element  xpath=//optional-menu/div/a[contains(., '${user}')]
+    Input Text  login_username  ${user}
+    Input Text  login_password  ${pw}
+    Click button  LOG IN
+    Wait Until Page Contains  Projects
+    Wait Until Keyword Succeeds  5x  1  Page Should Contain Element  xpath=/html/body/harbor-app/harbor-shell/clr-main-container[@class='main-container']/navigator/clr-header[@class='header-5 header']/div[@class='header-actions']/clr-dropdown[@class='dropdown bottom-right']/button[@class='nav-text dropdown-toggle']/span[contains(., '${user}')]
```

Comment:
@rogeliosanchez These are valid concerns, but I think these can be part of subsequent PR merges, as there is still more UI work for other test cases. I also want to have the UI change/test cycle to a minimum at this point.

In the meantime, it would be good to create a story for tagging IDs so we could track this with the Admiral team. Can you please create one and tag it to this PR?